src/tree/mod.rs

```diff
@@ -602,7 +605,17 @@ impl Tree {
         // NOTE: Create key hash for hash sharing
         // https://fjall-rs.github.io/post/bloom-filter-hash-sharing/
         #[cfg(feature = "bloom")]
-        let key_hash = crate::bloom::BloomFilter::get_hash(key.as_ref());
```

Comment:
When the prefix bloom filter is separated, this can all be reverted because the point read path does not change at all.